diff --git a/open_manipulator_bringup/launch/omx_f_follower_ai.launch.py b/open_manipulator_bringup/launch/omx_f_follower_ai.launch.py
--- a/open_manipulator_bringup/launch/omx_f_follower_ai.launch.py
+++ b/open_manipulator_bringup/launch/omx_f_follower_ai.launch.py
@@ -111,5 +111,5 @@ def generate_launch_description():
         parameters=[{'robot_description': urdf_file}, controller_manager_config],
         output='both',
         condition=UnlessCondition(use_sim),
-        remappings=[('/arm_controller/joint_trajectory', '/leader/joint_trajectory')],
+        remappings=[],
     )
